### PR TITLE
feat(single-animal): other animals section

### PR DESCRIPTION
**Pull request by AmandaKr**
## Jira issue

JIRA issue: [FSU-172](https://klau.atlassian.net/browse/FSU-172)

## Describe your changes
This PR introduces other animals section for single animal page with filtered animal data (we don’t display the current animal, only rest of them excluding adopted)
 [Example link](https://sterczaceuszy-git-fsu-172-other-animals-fsu.vercel.app/do-adopcji/wiosna) 👀 

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/frontend/app/lib/queries.js [hunk 1]
```diff
@@ -95,3 +95,24 @@ export const GET_ANIMAL = gql`
...
 `
+
+export const GET_OTHER_ANIMALS = gql`
+  query GetAnimal($slug: String) {
+    animalCollection(where: { slug_not: $slug }) {
+      total
+      items {
+        sys {
+          id
+        }
```
Comment (im-klau) [inline packages/frontend/app/lib/queries.js:106]: You don't need the `id` probably. I guess it's not needed anywhere in our queries, sorry, I forgot to remove it 🙈

File: packages/frontend/app/templates/Animal/Animal.jsx [hunk 2]
```diff
@@ -76,6 +78,7 @@ const Animal = ({ data }) => (
 
 Animal.propTypes = {
   data: animalType.isRequired,
+  otherAnimals: animalsType.isRequired,
```
Comment (im-klau) [inline packages/frontend/app/templates/Animal/Animal.jsx:81]: I'm afraid this prop type isn't the correct one, as the otherAnimals prop is an array:)

File: packages/frontend/app/lib/queries.js [hunk 3]
```diff
@@ -95,3 +95,21 @@ export const GET_ANIMAL = gql`
     }
   }
 `
+
+export const GET_OTHER_ANIMALS = gql`
```
Comment (im-klau) [inline packages/frontend/app/lib/queries.js:99]: Amanda - our new, GraphQL developer :D 
Well done 👏

**Review by im-klau — APPROVED** [review]

**Review by Ebmili — APPROVED** [review]
Looks great :)

**Pull request merged**